de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc

```diff
@@ -1193,9 +1193,72 @@ Now, in the _Navigator_ double click on ConfigurationTree and scroll to the Mass
 Here you can find a budget breakdown.
 |image:chapterModeling/MassConceptBudget.png[Insepcting the mass budegt breakdown]
 
+|===
+==== Functional Electrical Architecture
+===== Add Interface to the model
+Virtual Satellite enables the user to add _Interfaces_ to the system model. Add the concept *Functional Electrical Architecture* as described in section in <<Activating a Concept>>
+Elements of the functional electrical architecture can be added to the system model by opening the context menu with the right mouse and click. 
+
+There are _InterfaceEnds_ that are of a specific _InterfaceType_  and have the potential to connect _Interfaces_.
+
+_InterfaceTypes_  need to be defined as child elements of an _InterfaceTypeCollection_ .
+The _InterfaceTypeCollection_  is created by clicking with the right mouse on the repository and select menu:Context[Functional Electrical Architecture>Add InterfaceTypeCollection].
+image:chapterModeling/subsectionFEA/InterfaceType.png[InterfaceType_Collection]
+A type can be created by clicking with the right mouse on the _InterfaceTypeCollection_ and selecting menu:Context[Functional>Electrical Architecture>Add InterfaceType]
+image:chapterModeling/subsectionFEA/AddInterfaceType.png[InterfaceType]
+
+An _InterfaceType_ can be a _DataInterfaceType_ , a _ThermalReferencePoint_ or a _PowerInterfaceType_.
+
+By adding a _RangedVoltageDefinition_ or a _FixedVoltageDefinition_ to the _PowerInterfaceType_ a minimum and maximum voltage  can be specified.
+
+image:chapterModeling/subsectionFEA/VoltageSpecificationPowerInterfaceType.png[Specify the voltage of a power Interface]
+
+_InterfaceEnds_ should be defined as part of an ElementDefinition in a product tree.
+This definition enables elements of the configuration tree to inherit properties defined in the product tree.
+It is added to the product tree by clicking with the right mouse on the menu:Context[product tree >Functional Electrical Architecture>Add _Interface_End].

```

Comment:
The Add_Interface_End is deactivated, I think something should be missing.

Reply:
Done

Reply:
Is this really done? This looks like the same issue as with my other comments.
You cannot add an InterfaceEnd to a ProductTree - only to an ElementDefinition.
So selecting a product tree -> add -> Interface end will give you a deactivated menu.

Reply:
Well it was kind of done because the sentence over this one was explicitly stating that you have to add it to ElementDefinitions within the product tree... However, to make it more clear, I also improved this sentence know.